Lib/fontTools/varLib/mutator.py

```diff
@@ -48,8 +48,8 @@ def main(args=None):
 			coordinates += GlyphCoordinates(var.coordinates) * scalar
 		_SetCoordinates(varfont, glyphname, coordinates)
 
-	print("Removing GX tables")
-	for tag in ('fvar','avar','gvar'):
+	print("Removing variable tables")
+	for tag in ('fvar','avar','gvar','HVAR'):
```

Comment:
So leave it as it was is what you're saying, yes? Sorry I didn't know this was a WIP.
What's missing, BTW?